cpp/ql/lib/experimental/semmle/code/cpp/ir/dataflow/internal/DataFlowImpl3.qll

```diff
@@ -1169,8 +1169,23 @@ pragma[nomagic]
 private int branch(NodeEx n1, Configuration conf) {
   result =
     strictcount(NodeEx n |
-      flowOutOfCallNodeCand1(_, n1, _, n, conf) or flowIntoCallNodeCand1(_, n1, n, conf)
-    )
+        flowOutOfCallNodeCand1(_, n1, _, n, conf) or flowIntoCallNodeCand1(_, n1, n, conf)
+      ) + sum(ParamNodeEx p1 | | getLanguageSpecificFlowIntoCallNodeCand1(p1, n1, conf))
+}
+
+/**
+ * Gets an additional term that is added to `branch` and `join` when deciding whether
+ * the amount of forward or backward branching is within the limit specified by the
+ * configuration.
+ */
+pragma[nomagic]
+private int getLanguageSpecificFlowIntoCallNodeCand1(
+  ParamNodeEx p1, ArgNodeEx n1, Configuration conf
```

Comment:
Let's switch the order of the `n1` and `p1` columns to match the flow direction.

Reply:
Fixed in fc07c42bf6d8a1a0ae3c1c431322e6f8ad6f6512.